### PR TITLE
Run screenshot-tests in nightly CI

### LINKED ISSUE #3150: Run screenshot tests periodically or on-demand
Running the dataset rendering screenshot tests on my machine takes roughly 4 minutes and uses up all 800% of my CPU :sweat_smile: 

Ideally we would run the screenshot tests periodically or at least make it possible to execute them on demand **on another machine**.
To run the screenshot tests we need a deployed and up-to-date dev instance, e.g. `https://master.webknossos.xyz/`. The tests can then be run using the command URL=`https://master.webknossos.xyz/ docker-compose up screenshot-tests`.
I'm not sure whether it's possible to auto-deploy `https://master.webknossos.xyz/` every time the master changes? In that case we could create a simple cron job or circle CI [scheduled job](https://support.circleci.com/hc/en-us/articles/115015481128-Scheduling-jobs-cron-for-builds-) that runs the tests every night on weekdays or so.

**Alternatively**, we could execute the screenshot tests using a local webknossos instance, but then we would need the dev datasets for that.

**Or** we could create a slack command that executes the tests on one of our servers using a supplied URL of a specific wK instance. In that case, the user triggering the tests would make sure that the wK instance is up to date.

@normanrz or @jstriebel Any ideas or comments on the easiest approach here?

### PR DESCRIPTION
**Pull request by jstriebel**
### Steps to test:
- [CI "nightly" build](https://circleci.com/gh/scalableminds/webknossos/6016) is successful (currently hourly)

### TODO before merge:
- set workflow to `master` branch and actual nightly cron

### Issues:
- fixes #3150

------
- [ ] ~~Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
- [ ] ~~Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- [ ] ~~Updated [documentation](../blob/master/docs) if applicable~~
- [ ] ~~Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -156,3 +198,13 @@ workflows:
...
+  nightly:
+    jobs:
+      - nightly
+    triggers:
+      - schedule:
+          cron: "15 * * * *"
+          filters:
+            branches:
+              only:
+                - nightly-ci
```
Comment (normanrz) [inline .circleci/config.yml]: This is only for testing the PR?

Comment (jstriebel) [reply .circleci/config.yml]: yes, I will change this to `master` before merging, as well as the cron

Comment (philippotto) [reply .circleci/config.yml]: This would change to `master`, right?

**Comment by jstriebel** [comment]
@normanrz can you merge this? I can't, cause of the last skip ci^^

**Comment by daniel-wer** [comment]
@jstriebel Nice PR :) Where are we (or who is) getting notified if the screenshot tests fail?

**Comment by jstriebel** [comment]
@daniel-wer We get notifications via Slack, and directly from CircleCI via Email (I guess if you are registered via github in CirlceCI and follow the project: https://circleci.com/add-projects/gh/scalableminds).

**Pull request merged**